app/javascript/widget/components/ChatFooter.vue

```diff
@@ -133,6 +133,13 @@ export default {
 <style scoped lang="scss">
 @import '~widget/assets/scss/variables.scss';
 
+.footer-chat {
+  @media screen and (min-width: 800px) {
+    width: 50%;
+    margin: 0 auto;
+  }
```

Comment:
I think that it works better with these settings
```suggestion
  max-width: 800px;
  margin: 0 auto;
```

By doing this, we avoid the breakpoint moment where the elements rearrange because of the media queries. This fix works for the 3 files you changed, except for `ConversationWrap`, there you have to add a `width: 100%` prop 

<details><summary><b>Video</b></summary><br>

https://user-images.githubusercontent.com/22230419/220791875-c91baa8f-a3f0-419c-a52e-6c995d732b40.mov

</details>